FlipperZero/Shared/Assets.xcassets/LaunchScreenColor.colorset/Contents.json

```diff
@@ -0,0 +1,20 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "display-p3",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0.263",
+          "green" : "0.576",
+          "red" : "0.935"
```

Comment:
The screen itself looks fine, but on the device it just flickers with orange. I think we better use light/dark background color for now

Reply:
It is because startup is too fast. :) Actually logo/picture launch screens are against HIG but no-one was ever banned from App Store for this reason. It is better to match it with the app's main screen when it would be closer for release.